lldb/source/Target/Target.cpp

```diff
@@ -2251,12 +2251,16 @@ Target::GetScratchTypeSystemForLanguage(lldb::LanguageType language,
       }
     } else if (create_on_demand) {
       if (StreamSP error_stream_sp = GetDebugger().GetAsyncErrorStream()) {
+        auto module = GetExecutableModule();
+        const char *module_name = nullptr;
+        if (module_name)
```

Comment:
That looks like a trivially false condition?

Reply:
Sigh, that should be module :(

Reply:
as in `if (module)`